packages/api/tests/api/contributions.test.ts

```diff
@@ -0,0 +1,163 @@
+import { Collection } from '@mikro-orm/core';
+import { contributions } from '../../src/api/contributions';
+import { Contribution } from '../../src/entities/Contribution';
+import { User } from '../../src/entities/User';
+import logger from '../../src/logger';
+import { testHandler } from '../testUtils/testHandler';
+
+const sampleUser: Partial<User> = {
+  name: 'Bill Nye',
+  pronouns: 'he/him',
+  schoolName: 'Science School',
+  assign: jest.fn(),
+  isAdmin: false,
+  githubId: '234234',
+  id: '1',
+  contributionList: new Collection<Contribution>(this),
+};
+
+const sampleUserNoContributionsList: Partial<User> = {
+  name: 'Bill Smith',
+  pronouns: 'he/him',
+  schoolName: 'School',
+  assign: jest.fn(),
+  isAdmin: false,
+  githubId: '0987654321',
+  id: '2',
+};
```

Comment:
Because we're tricking Jest into believing the type of this is something specific, we really should ONLY be including data that's relevant to the test. Can you remove any fields (aside from `id`) that aren't being used in tests and just return that object instead?